components/kyma-environment-broker/internal/broker/instance_get.go

```diff
@@ -68,5 +73,16 @@ func (b *GetInstanceEndpoint) GetInstance(_ context.Context, instanceID string,
 			Labels: ResponseLabels(*op, *instance, b.config.URL, b.config.EnableKubeconfigURLLabel),
 		},
 	}
+
+	if b.config.ShowTrialExpireInfo {
+		// for test purposes on DEV env
+		if instance.GlobalAccountID == trialGAForTests && instance.SubAccountID == trialSAForTests {
```

Comment:
nit: the plan is to later remove this filter for a specific GA and SA? given it's behind a feature flag, I'd vote for showing when the flag is on (that way we can control whether it is on or off per env)

Alternatively, there could be one more flag of string type taking comma separated list of `GA:SA` for fine-grained control over the roll out?

Reply:
GA and SA are only temporarily placed in the code, I needed to specify my own runtime to limit the change to only one instance on DEV.